gTile@shuairan/files/gTile@shuairan/extension.js

```diff
@@ -323,6 +323,11 @@ function move_resize_window(metaWindow, x, y, width, height) {
   metaWindow.move_frame(true, x, y);
 }
 
+function getHeight(panel) {
```

Comment:
This should be `getPanelHeight` now it works for me.
```suggestion
function getPanelHeight(panel) {
```